index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MyPortofolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+
+</head>
+
+<body>
+    <header class="navbar container fixed-top">
+        <a href="" id="logo">
+            Welcome
+        </a>
+        <nav class="menu">
+            <i class="fa-solid fa-bars "></i>
+        </nav>
+    </header>
+    <section class="header">
+        <div class="container">
+            <h1 class="identity">
```

Comment:
- [ ] Good job creating your first project at microverse  😄 , And fortunately it looks good 👏 ,  however your page does not have a background-image ,  make sure the background image you use that is used in the Figma 👍 

|  Your Design |  Figma Design |
| :---: | :---: |
| ![Capture 2](https://user-images.githubusercontent.com/80458199/228266836-27e4c0a2-fa4b-4149-ae23-b5f44692a642.PNG) |  ![Capture](https://user-images.githubusercontent.com/80458199/228266911-28b42358-f73a-4191-aa14-2fcdc0b79a0b.PNG)|